pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -146,7 +146,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundHTTPRouteConfig(env *m
 		for _, host := range virtualHostWrapper.VirtualServiceHosts {
 			virtualHosts = append(virtualHosts, route.VirtualHost{
 				Name:    fmt.Sprintf("%s:%d", host, virtualHostWrapper.Port),
-				Domains: []string{host},
+				Domains: []string{host, fmt.Sprintf("%s:%d", host, virtualHostWrapper.Port)},
```

Comment:
shouldn't this be backend port? if "google.com:80" is bound to "backend:8080" which port is used here?

Reply:
I hope the first (:80) - backend port should only be reflected in EDS and the inbound listener/cluster, not routes.

Reply:
It’s front end port. This is what curl appends when you do host:port